docs/events/timeline-ui-overview.asciidoc

```diff
@@ -7,30 +7,43 @@ You can add alerts from multiple indices to a Timeline to facilitate advanced in
 You can drag or send fields of interest to a Timeline to create the desired query. For example, you can add fields from tables and histograms
 on the *Overview*, *Alerts*, *Hosts*, and *Network* pages, as well as from
 other Timelines. Alternatively, you can add a query directly in Timeline
-by clicking *+ Add field*.
+by expanding the <<narrow-expand,query builder>> and clicking **+ Add field**.
 
 [role="screenshot"]
 image::images/timeline-ui-updated.png[example Timeline with several events]
 
-To avoid losing your changes, you must save the Timeline before moving to a different {security-app} page.
-
-If you change an existing Timeline, you can use the **Save as new timeline** toggle to make a new copy of the Timeline, without overwriting the original one.
-
-TIP: You can record and share your findings with others by attaching your Timeline to a
-<<cases-overview, case>>.
-
 In addition to Timelines, you can create and attach Timeline templates to
 <<detection-engine-overview, detection rules>>. Timeline templates allow you to
 define the source event fields used when you investigate alerts in
 Timeline. You can select whether the fields use predefined values or values
 retrieved from the alert. For more information, refer to <<timeline-templates-ui>>.
 
+[discrete]
+[[open-create-timeline]]
+
+== Create new or open existing Timeline
+
+To make a new Timeline, choose one of the following:
+
+* Go to **Timelines**, then click **Create new Timeline**. 
+* Go to the Timeline bar (which is at the bottom of most pages), click the image:images/add-new-timeline-button.png[Click the add new button,20,20] button, then click **Create new Timeline template**.
+* From an open Timeline or Timeline template, click **New** -> **New Timeline**.
+
+To open an existing Timeline, choose one of the following: 
+
+* Go to the Timelines page, then click a Timeline's title. 
+* Go to the Timeline bar, click the image:images/add-new-timeline-button.png[Click the add new button,20,20] button, then click **Open Timeline**.
+* From an open Timeline  or Timeline template, click **Open**, then select the appropriate Timeline. 
+
+To avoid losing your changes, you must save the Timeline before moving to a different {security-app} page. If you change an existing Timeline, you can use the **Save as new timeline** toggle to make a new copy of the Timeline, without overwriting the original one.
```

Comment:
```suggestion
To avoid losing your changes, save the Timeline before moving to a different {security-app} page. If you change an existing Timeline, you can use the **Save as new timeline** toggle to make a new copy of the Timeline without overwriting the original one.
```